src/content/structured/components/cards/code.mdx

```diff
@@ -760,9 +1112,17 @@ export const wrapped = [
     heading="Card heading"
     subheading="Card subheading" 
     message="This is an example of a card component wrapped by a link.">
-    <SlottedSVG slot="icon" ... >
-      ...
-    </SlottedSVG>
+  ]<svg
```

Comment:
```suggestion
  <SlottedSVG
```

Reply:
rolled back changes outside scope of ticket- this no longer applies.